CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Honeycomb Lambda Extension Changelog
 
+## [v10.3.0] Layer version 10 - 2022-07-25
+
+- Fixes OpenSSL CVE | [@pkanal](https://github.com/pkanal)
```

Comment:
We're not really consistent throughout our repos, but I'd love to see this separated into an `### Added` section with the added regions, and `###Maintenance` section with the OpenSSL CVE fix.